src/main/java/seedu/address/model/person/UniquePersonList.java

```diff
@@ -134,4 +135,8 @@ private boolean personsAreUnique(List<Person> persons) {
         }
         return true;
     }
+
+    public void sort() {
```

Comment:
I like how concise it is. however, for future extensions on sorting, we can make the Comparator a variable so that we may sort based on other attributes too.